### PR TITLE
[3308][15.0][MIG] website_sale_wishlist_menuitem: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3308](https://www.quartile.co/web#id=3308&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Two things:

- Can we rename the module to website_sale_wishlist_menuitem
- Do you think it's a good idea to put this module in oca/e-commerce?

**Comment by AungKoKoLin1997** [comment]
> Do you think it's a good idea to put this module in oca/e-commerce?

@yostashiro  Yes. I think it's a good idea.

File: website_sale_wishlist_menuitem/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+* This module adds a menuitem for internal user to check all the wishlist products.
```
Comment (yostashiro) [inline website_sale_wishlist_menuitem/readme/DESCRIPTION.rst]:
```suggestion
* Adds a menuitem for internal users to be able to check/manage the wishlist content in the backend.
```

File: website_sale_wishlist_menuitem/views/product_wishlist_views.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <record id="action_product_wishlist" model="ir.actions.act_window">
+        <field name="name">Product Wishlist</field>
+        <field name="res_model">product.wishlist</field>
+        <field name="view_mode">tree</field>
+        <field name="search_view_id" ref="product_wishlist_search" />
+    </record>
+    <menuitem
+        id="menu_product_wishlist"
+        name="Product Wishlist"
+        parent="website_sale.menu_catalog"
```
Comment (yostashiro) [inline website_sale_wishlist_menuitem/views/product_wishlist_views.xml]:
```suggestion
        parent="website_sale.menu_orders"
```
I think this node would be more appropriate.

File: website_sale_wishlist_menuitem/i18n/ja.po
Comment (yostashiro) [inline website_sale_wishlist_menuitem/i18n/ja.po:1]: Please regenerate the translation file, and apply the same translation.  Please ask @rinaldifirdaus to support if you have a difficulty.

File: website_sale_wishlist_menuitem/views/product_wishlist_views.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                        domain=""
+                        context="{'group_by':'product_id'}"
+                    />
+                    <filter
+                        string="Customer"
+                        name="customer"
+                        domain=""
+                        context="{'group_by':'partner_id'}"
+                    />
+                </group>
```
Comment (yostashiro) [inline website_sale_wishlist_menuitem/views/product_wishlist_views.xml:35]: Let's move this to the bottom inside the search tag, following the convention.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]
Code + functional reviews. 👍 

@AungKoKoLin1997 Please go ahead and create a PR in oca/e-commerce.

**Pull request merged**